src/familybot/web/templates/admin.html

```
@@ -1,58 +1,109 @@
+            <div class="grid-2" style="align-items:start">
+
+                <!-- Left column: commands -->
+                <div style="display:flex;flex-direction:column;gap:16px">
+
+                    <!-- Database -->
+                    <div class="card">
+                        <div class="card-header">
+                            <div class="card-title"><i class="fas fa-database"></i> Database</div>
+                        </div>
+                        <div class="card-body" style="display:flex;flex-direction:column;gap:10px">
+                            <button class="btn btn-primary admin-btn" data-cmd="populate" data-endpoint="/api/admin/populate-database" data-method="POST">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)